src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java

```diff
@@ -10,28 +10,23 @@
 import org.springframework.security.oauth2.jwt.Jwt;
 import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.GenericContainer;
 import uk.gov.hmcts.darts.test.common.LogUtil;
 import uk.gov.hmcts.darts.test.common.MemoryLogAppender;
 import uk.gov.hmcts.darts.testutils.stubs.DartsDatabaseStub;
 
 import java.util.List;
 
 /**
- * Base class for integration tests running with H2 in PostgreSQL compatibility mode.
+ * Base class for integration tests running with H2 in Postgres compatibility mode.
```

Comment:
Not sure this comment is needed here?

Reply:
I have this comment to distinguish this class with `PostgresIntegrationBase` which has the following comment:
```
/**
 * Base class for integration tests running against a containerized Postgres with Testcontainers.
 */
```
so it's clear when to use `PostgresIntegrationBase` vs `IntegrationBase`. 
But I can remove it if it's unnecessary

Reply:
Ah ok, I get it, that's cool. Is it worth mentioning that using this class runs redis as well?

Reply:
I have noted this down and will change the comment as part of a next PR of mine. So I don't have to wait for a new build